docs/src/network.md

```diff
@@ -791,6 +791,28 @@ var context = await browser.NewContextAsync(new() {
 await context.CloseAsync();
 ```
 
+### Recording HAR from a test
+
+To record a HAR file we use the `routeFromHAR` method. This method takes in the path to the HAR file and an optional object of options.
+
+The options object can contain the URL so that only requests with the URL matching the specified glob pattern will be served from the HAR File. If not specified, all requests will be served from the HAR file.
+
+The `update` option updates the given HAR file with the actual network information instead of serving from the file. In order to record the HAR file, you need to set `update` to true.
+
+```js
+test('Record a HAR', async ({
```

Comment:
We don't usually split test header into multiple lines in other examples. Let's be consistent.